src/mesh/p4est_mesh.jl

```diff
@@ -139,22 +226,61 @@ function connectivity_structured_periodic(cells_x, cells_y)
   vertices = C_NULL
```

Comment:
I just see this now... why use `num_` instead of `n_`, which is so far the convention in Trixi?

(NB: This comment is attached to the wrong line since I cannot comment on the correct one above)

Reply:
I named the variables consistently with the parameters of the function `p4est_connectivity_new_copy`.

Reply:
Hm, I'd prefer the use of names that match our conventions.